submissions/adeyemiJuwon/database/init.sql

```diff
@@ -0,0 +1,210 @@
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `admin_users`
+--
+
+
+INSERT INTO `admin_users` VALUES (1,'juwon','samju7778@gmail.com','ade100','23480-667-76598'),
+(2,'samuel','samju6677@gmail.com','sam100','23480-667-76598'),
+(3,'nuru','nuru@gmail.com','nuru100','23480-667-87598'),
+(4,'itunu','itunu@gmail.com','itunu100','23480-667-66598'),
+(5,'feyi','feyi@gmail.com','feyi100','23480-567-76598'),
+(6,'temidayo','temi@gmail.com','temi100','23480-667-76598'),
+(7,'seye','seye@gmail.com','seye100','23480-667-76690');
+
+
+--
+-- Table structure for table `categories`
+--
+
+DROP TABLE IF EXISTS `categories`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `categories` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `category` varchar(100) NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `categories`
+--
+
+
+INSERT INTO `categories` VALUES (1,'Laptop Accessiories '),
+(2,'Baby Care & Wear'),
+(3,'Women Collection'),
+(4,'Electronic Accessiories'),
+(5,'Smart Home & Garden'),
+(6,'Lights & Lighting');
+
+--
+-- Table structure for table `customers`
+--
+
+DROP TABLE IF EXISTS `customers`;
+CREATE TABLE `customers` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `first_name` varchar(50)  NOT NULL,
+  `last_name` varchar(50) NOT NULL,
+  `email_address` varchar(250) NOT NULL,
+  `password` varchar(250)  NOT NULL,
+  `phone` varchar(50)  NOT NULL,
+  `contact_address` varchar(250) NOT NULL,
+  `street_address` varchar(250)  NOT NULL,
+  `zip_code` int(11) DEFAULT NULL,
+  `city` varchar(50)  NOT NULL,
+  `state` varchar(50) NOT NULL,
+  `country` varchar(50)  NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `customers`
+--
+
+
+INSERT INTO `customers` VALUES (1,'Samuel','Adeniyi','ade@gmail.com','adew200',' 234-80775-67589','5 Ogunmefun Somolu Lagos','Ogunmefun',27704,'Lagos','Lagos','Nigeria'),
+(2,'Ronke','Adeyemi','ronke@gmail.com','ronke200','234-80665-67589','10 Ring Road Ibadan Oyo','Ring Road',27705,'Ibadan','Oyo','Nigeria'),
+(3,'Itunu','Adekoya','itunu@gmail.com','itunu200','234-80675-67589','11 wale Somolu Lagos','wale',27704,'Lagos','Lagos','Nigeria'),
+(4,'Wale','Ejigboye','wale@gmail.com','wale200','234-80775-67579','5 Ogonbiade Kunlede Ilorin ','Ogonbiade',27703,'Ilorin','Kwara','Nigeria');
+
+
+--
+-- Table structure for table `order_items`
+--
+
+DROP TABLE IF EXISTS `order_items`;
+CREATE TABLE `order_items` (
+  `order_id` int(11) NOT NULL,
+  `product_id` int(11) NOT NULL,
+  `quantity` int(11) NOT NULL,
+  `unit_price` decimal(8,2) NOT NULL,
+  `total_amount` decimal(8,2) NOT NULL,
+  PRIMARY KEY (`order_id`,`product_id`),
```

Comment:
columns references another field should not be made are already `INDEX` they shouldn't be made `PRIMARY KEY`